base/strings/io.jl

```diff
@@ -57,6 +57,8 @@ end
 Print (using [`print`](@ref)) `xs` followed by a newline.
 If `io` is not supplied, prints to [`stdout`](@ref).
 
+See also: [`show`](@ref), [`@show`](@ref), [`@info`](@ref Logging), [`sprint`](@ref), [`Printf.@printf`](@ref man-printf).
```

Comment:
```suggestion
```

I don't think any of these are really that related to println just because they are alternatives to IO

Reply:
This is too many, but perhaps a few things people who know the name of `println` might be wish they knew about? 
```suggestion
See also: [`@show`](@ref), [`@info`](@ref Logging), [`Printf.@printf`](@ref Printf).
```